server/serverbase.js

```diff
@@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
       }
     }
   }
+  function loadThumb() {
+    return function(req, res, next) {
+      req.url = path.dirname(req.url) + '/.thumbs/' +
+                path.basename(req.url);
+      staticUserData(req, res, next);
```

Comment:
So you refer to a thumb by saying /thumbs/projectname.png, is that right?  So for example, if somebody uploads a png to their project, if we created a thumb it would be /thumbs/myimage.png.png.